tensorflow/python/ops/gradient_checker_v2.py

```diff
@@ -313,6 +313,17 @@ def compute_gradient(f, x, delta=1e-3):
 
   Raises:
     ValueError: If result is empty but the gradient is nonzero.
+    ValueError: If x is not list, but any other type.
+
+  Example:
+  ```python
+  @tf.function
+  def test_func(x):
+    return x*x
+
+  theoretical, numerical = tf.test.compute_gradient(test_func, [1.0])
+  theoretical, numerical
```

Comment:
Maybe this last line should have an assertion that theoretical and numerical are close?

Reply:
Hi Alex, thanks for the review!

would it help if I print the theoretical and numerical results?
```suggestion
  theoretical, numerical
  
#  ((array([[4.]], dtype=float32),), (array([[3.9999483]], dtype=float32),))
```

Not sure if testing would be that useful in the documentation. Since it's intended for a user, not test runner. What do you think?

Reply:
Alternatively, I can also add something like 

```python 
from tensorflow.python.ops import gradient_checker_v2 as gradient_checker

error = gradient_checker.max_error(theoretical, numerical)
print("error = %f" % error)
# error = 0.000052

assert error < 1e-4
```

Just not sure if it's not too much for a docs example :)

Reply:
I like the suggestion of printing the values; better than importing internal modules.